Guidance that indicate to avoid sharing types between saga types (#5591)
Particular/docs.particular.net: nservicebus/sagas/index.md

snippet: simple-saga-data

### Avoid sharing types between sagas

Saga data types should not be shared across different sagas. Sharing types can result in persisters physically sharing the same storage structure which should be avoided.

> WARNING: Sharing property types should also be avoided. Depending on the persister these could could also result in shared storage structure.

Author: ```suggestion
WARNING: Sharing property types should also be avoided. Depending on the persister implementation sharing property types can result in shared storage structure.
```
